roles/internal/webserver-app/tasks/drupal.yml

```diff
@@ -106,3 +106,20 @@
     dest: "{{ drupal_external_libraries_directory }}/pdf.js"
     creates: "{{ drupal_external_libraries_directory }}/pdf.js/build"
     remote_src: yes
+
+- name: Add project's /vendor/bin to $PATH
+  lineinfile:
+    path: ~/.profile
+    line: 'PATH="$PATH:{{ webserver_document_root }}/drupal/vendor/bin"'
+    state: present
+  become: false
+  when: ansible_os_family == "Debian"
+
+- name: 
```

Comment:
Can we add a description to the name property here like the one above? By convention you can just copy the other and add the OS it pertains to in parentheses.

Reply:
Otherwise this looks good.

Reply:
how's that?  e6d38ab